lib/opentelemetry/trace/exporter/console_span_exporter.lua

```diff
@@ -0,0 +1,43 @@
+-------------------------------------------------------------------------------
```

Comment:
We can simplify the filename `console_span_exporter.lua` to `console.lua`.

Reply:
👍 f3d5607deeccd910eb5acd4864735d9d24042fd3